spec/units-of-measure.md

```diff
@@ -1,138 +1,133 @@
 # Units of Measure
 
-F# supports static checking of _units of measure_. Units of measure, or _measure_ s for short, are like
-types in that they can appear as parameters to other types and values (as in `float<kg>`, `vector<m/s>`,
-`add<m>`), can contain variables (as in `float<'U>`), and are checked for consistency by the type-checker.
+F# supports static checking of _units of measure_. Units of measure, or _measures_ for short, are like types in that they can appear as parameters to other types and values (as in `float<kg>`, `vector<m/s>`, `add<m>`), can contain variables (as in `float<'U>`), and are checked for consistency by the type-checker.
```

Comment:
Shouldn't it say "can be represented by variables" rather than "contain variables"?
(This is a comment about the original, not about the change.)